client/js/widgets/widget.js

```diff
@@ -1848,7 +1850,10 @@ export class Widget extends StateManaged {
           let nextTurnIndex = 0;
 
           if(a.turnCycle == 'position') {
-            if(a.turn == 'first') {
+            if (turnIDIndex !== -1) {
+              nextTurnIndex = c.findIndex(w => w.get('id') === a.turn);
```

Comment:
Isn't `turnIDIndex` already the value you want to set `nextTurnIndex` to?

Reply:
Fixed.